dockers/docker-dhcp-relay/docker_init.sh

```diff
@@ -2,15 +2,21 @@
 
 # Generate supervisord config file
 mkdir -p /etc/supervisor/conf.d/
-sonic-cfggen -d -t /usr/share/sonic/templates/docker-dhcp-relay.supervisord.conf.j2 > /etc/supervisor/conf.d/docker-dhcp-relay.supervisord.conf
 
-# Generate the script that waits for all interfaces to come up and make it executable
-sonic-cfggen -d -t /usr/share/sonic/templates/wait_for_intf.sh.j2 > /usr/bin/wait_for_intf.sh
-chmod +x /usr/bin/wait_for_intf.sh
+# Generate scripts that:
+# 1. dhcp-relay supervisord configuration, 
+# 2. waits for all interfaces to come up, and
+# 3. port alias map
```

Comment:
Not all of these are scripts. Suggest rewording:

```
# Generate the following files from templates:
# 1. supervisord configuration 
# 2. wait_for_intf.sh, which waits for all interfaces to come up
# 3. port-to-alias name map
```

Reply:
done